iphone/Classes/TiUIiPhoneTableViewSeparatorStyleProxy.m

```diff
@@ -16,9 +16,17 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.TableViewSeparatorStyle";
 }
 
-MAKE_SYSTEM_PROP(NONE,UITableViewCellSeparatorStyleNone);
-MAKE_SYSTEM_PROP(SINGLE_LINE,UITableViewCellSeparatorStyleSingleLine);
+-(NSNumber*)SINGLE_LINE
+{
+    DEPRECATED_REPLACED(@"Ti.UI.iPhone.TableViewSeparatorStyle.SINGLE_LINE", @"5.2.0", @"Ti.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE");
```

Comment:
Needs to be without the "Ti." prefix that is concatenated inside the macro (same thing with the other line). Greets!